substrate/frame/staking/src/tests.rs

```diff
@@ -5308,6 +5317,7 @@ fn min_commission_works() {
 
 #[test]
 #[should_panic]
+#[cfg(debug_assertions)]
```

Comment:
Because they are failing if not running with debug asserts enabled. Aka if you just run `cargo test --all --release`. In CI we also enable debug asserts in release mode.